src/sonic-config-engine/sonic-cfggen

```diff
@@ -244,7 +244,10 @@ def main():
         print(template.render(data))
 
     if args.var_json != None:
-        print(json.dumps(FormatConverter.to_serialized(data[args.var_json]), indent=4, cls=minigraph_encoder))
+        try :
+            print(json.dumps(FormatConverter.to_serialized(data[args.var_json]), indent=4, cls=minigraph_encoder))
+        except KeyError:
+            return
```

Comment:
better to write like following:

```
if not data.has_key(args.var_json):
   print "{}"
else:
   print(json.dumps(FormatConverter.to_serialized(data[args.var_json]), indent=4, cls=minigraph_encoder))
```

Reply:
if there is no key, we should still print out a valid empty json string, which is {}

Reply:
Thank you for your comments. I have added the code changes based on your suggestion.

Reply:
Request to review changes.